File: SMM.IServices/Models/User/UserModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SMM.IServices.Models.User
{
   public class UserModel
    {
        public int Id { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public int? VkId { get; set; }
        public string OkId { get; set; }
        public string ImageUrl { get; set; }
    }
}

File: SMMService/asserts/js/User/UserModel.js
var User = User || {};

(function () {

    User.UserModel = function (theParams) {
        theParams = theParams || {};
        this.Id = ko.observable(theParams.Id || 0);
        this.FirstName = ko.observable(theParams.FirstName || "");
        this.LastName = ko.observable(theParams.LastName || "");
        this.ImageUrl = ko.observable(theParams.ImageUrl || "");
        return this;
    };

    /**
    * Определяем конструктор
    */
    User.UserModel.prototype.constructor = User.UserModel;

  
})();